coremltools/converters/nnssa/commons/builtins/type_bool.py

```diff
@@ -42,3 +42,6 @@ def __double__(self):
     @annotate(delay_type.str)
     def __str__(self):
         return str(self.val)
+
+def is_bool(t):
+    return t is bool or isinstance(t,bool)
```

Comment:
What's the purpose of checking for both type equivalency and type membership in the same function? This behavior seems strange:

```
is_bool(bool) # True
is_bool(True) # True
is_bool(False) # True
is_bool(literally_anything_else) # False
```

This function also doesn't appear to be used anywhere.